src/sql/workbench/contrib/profiler/browser/profilerEditor.ts

```diff
@@ -590,6 +591,8 @@ export class ProfilerEditor extends EditorPane {
 					this._updateSessionSelector();
 				} else {
 					this._profilerTableEditor.loadingSpinner.loading = false;		// Remove the loading indicator when the complete file is read
+					this._profilerTableEditor._overlay.style.paddingTop = '0px';	// Remove the added padding to keep the table header columns resize functional
```

Comment:
It seems like the reason this is happening is that the overlay is ALWAYS there - it just doesn't show anything when the loading spinner isn't active (but it still blocks clicks). This makes sense since the padding makes it take up space (and the width is set to 100%)

Did you verify that after this fix you can still interact with the other elements on the screen that are above the headers such as the dropdowns? Let's make sure this isn't just moving the div to block some other input.

Reply:
Yes, the other screen elements are still functional after the change.
So if the file is huge, the loading indicator keeps on spinning and that is the time when the headers aren't resizable. But the other elements above the header are.
![image](https://github.com/microsoft/azuredatastudio/assets/57200045/736b583a-59ef-46f9-9efa-52581474eaf6)

Once the complete file is loaded, the loading indicator disappears and the header becomes resizable again.

Reply:
Actually, you should just be able to use the top property instead of adding padding to have the loading spinner where you want while not blocking the UI at all. That might be a better solution.

Reply:
(that being said, if you can't get it working due to other conflicting properties or something then it's probably fine to just leave it like this. Not being able to resize while loading is likely fine for this scenario)

Reply:
top works. I didn't know that could be used in this manner too. 
Updating the code accordingly.
(To add, headers are resizable when the loading indicator is there. Also, all the other elements work as expected.)